__tests__/unit.test.js

```diff
@@ -0,0 +1,60 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const fs = require("fs");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/topics", () => {
+  test('GET: 200 - responds with an array of topic objects with "slug" and "description" properties', () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topic)).toBe(true);
+        body.topic.forEach((topic) => {
+          expect(topic).toEqual({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET: 404 - responds with an error message for all non-existent routes", () => {
+    return request(app)
+      .get("/api/not-a-route")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
+
+describe("GET /api", () => {
+  test("GET: 200 - responds with an object describing all the available endpoints on your API ", () => {
+    const endPoints = JSON.parse(fs.readFileSync(`endpoints.json`));
```

Comment:
Rather than having to use readfile, we can just require the .json file into the tests, then expect the response to equal the contents of the file. I'll leave it up to you if you wanna refactor it 🙂